Rules/CommonScripts/VoteCommon.as

```diff
@@ -221,13 +221,26 @@ void Vote(VoteObject@ vote, CPlayer@ p, bool favour)
 		{
 			vote.current_no++;
 		}
-
-		if (CanPlayerVote(vote, getLocalPlayer()) || getNet().isServer()) //include all in server logs
+		
+		bool should_show_votes;
+		CPlayer@ player = getLocalPlayer();
+		if (player is null)
 		{
-
-
-			client_AddToChat(getTranslatedString("--- {USER} Voted {DECISION} ---").replace("{USER}", p.getUsername()).replace("{DECISION}", getTranslatedString(favour ? "In Favour" : "Against")), vote_message_colour());
+			should_show_votes = getNet().isServer();
+		}
+		else
+		{
+			should_show_votes = (player.isDev() || player.isGuard() || getNet().isServer()
+			|| getSecurity().checkAccess_Feature(player, "admin_color") || player.isRCON());
```

Comment:
Wouldn't it make more sense to check for player.isMod() here instead of admin_color?

Reply:
Yep, I'll fix that